tendrl/node_agent/manager/manager.py

```diff
@@ -227,6 +228,34 @@ def on_pull(self, raw_data):
                     node_id=raw_data["node_id"],
                 )
             )
+        if "disks" in raw_data:
+            try:
+                self.etcd_client.delete(
+                    ("nodes/%s/Disks") % raw_data["node_id"], recursive=True)
+            except Exception as ex:
```

Comment:
You are catching the broadest python exception "Exception" and simply logging it and it gets suppressed , this is not allowed usually, read more here 

https://realpython.com/blog/python/the-most-diabolical-python-antipattern/